### ISSUE TITLE
Revisit VB Range converter to not require IsMacroType=true

### ISSUE THREAD

**Issue by govert** [Excel-DNA/Registration#18]
Use `xlSheetNm` and explicit address construction instead of `xlfReftext`.

**Comment by govert** [comment]
```
        Shared Function ReferenceToRange(reference As ExcelReference) As Range
            Dim CurrentExcelApplication As Microsoft.Office.Interop.Excel.Application
            CurrentExcelApplication = ExcelDnaUtil.Application

            Dim sheetName As String = XlCall.Excel(XlCall.xlSheetNm, reference)
            Dim index As Integer = sheetName.LastIndexOf("]")
            sheetName = sheetName.Substring(index + 1)
            Dim ws As Worksheet = CurrentExcelApplication.Sheets(sheetName)
            Dim target As Range = CurrentExcelApplication.Range(ws.Cells(reference.RowFirst + 1, reference.ColumnFirst + 1), _
                                                                 ws.Cells(reference.RowLast + 1, reference.ColumnLast + 1))

            Return target
        End Function
```

**Comment by govert** [comment]
Fixed with 0c7d4e940562693240b74e0fa02cb1aeaffc76a7
